### PR TITLE
Feature add/remove lote to wishlist

### PR DESCRIPTION
**Pull request by mikeyrayvon**
add functionality to add & remove lotes from user wishlist on firebase and app state

### CODE REVIEW

File: src/reducers/lotesReducer.js [hunk 1]
```diff
@@ -23,13 +23,22 @@ export default function loteReducer(state = initialState, action) {
           map(id => {
             const { title, artista, obras, price } = action.data[id];
 
+            let isWishlist = false;
```
Comment (varffs) [inline src/reducers/lotesReducer.js]: I don't think we need this bool anymore right? As there is another check in the header component.

File: src/reducers/member.js [hunk 2]
```diff
@@ -17,16 +18,34 @@ export default function userReducer(state = initialState, action) {
...
     }
-    case 'USER_DETAILS_UPDATE': {
-      if (action.data) {
+    case 'USER_WISHLIST_REMOVE': {
+      if (action.removeLote) {
+        const wishlist = state.wishlist;
+
+        // lodash func removes from array by value
+        const newWishlist = remove(wishlist, (lote) => {
+          return lote !== action.removeLote;
```
Comment (varffs) [inline src/reducers/member.js:27]: what is this condition about?

**Comment by varffs** [comment]
Looks good to me but @mixcas should check too. I think we do the optimistic stuff as another feature after this is in master.

**Comment by varffs** [comment]
Just tested this and it is a bit slow on my local (but my local is soooo slow now). Not sure if others are having that but will def need optimistic updates in the future

File: src/reducers/lotesReducer.js (hunk 1 above)
Comment (mikeyrayvon) [reply src/reducers/lotesReducer.js]: oh yea none of this should be here. isWishlist has nothing to do with lotesReducer

Comment (mikeyrayvon) [reply src/reducers/lotesReducer.js]: maybe would've have made more sense to do the check here. but i didn't do it this way

File: src/reducers/member.js (hunk 2 above)
Comment (mikeyrayvon) [reply src/reducers/member.js:27]: this function goes through the wishlist and returns a new array with everything not equal to the lote we want to remove

**Comment by mixcas** [comment]
Good to go, @mikeyrayvon can u resolve conflict and merge?

**Pull request merged**